app/lib/actions/katello/repository/errata_mail.rb

```diff
@@ -4,17 +4,16 @@ module Repository
       class ErrataMail < Actions::EntryAction
         middleware.use Actions::Middleware::ExecuteIfContentsChanged
 
-        def plan(repo, last_updated = nil, contents_changed = nil)
-          last_updated ||= repo.repository_errata.order('updated_at ASC').last.try(:updated_at) || Time.now
-          plan_self(:repo => repo.id, :last_updated => last_updated.to_s, :contents_changed => contents_changed)
+        def plan(repo, contents_changed = nil)
+          plan_self(:repo => repo.id, :contents_changed => contents_changed)
         end
 
         def run
           ::User.current = ::User.anonymous_admin
-
           repo = ::Katello::Repository.find(input[:repo])
+          last_updated = repo.repository_errata.order('updated_at ASC').last.try(:updated_at) || Time.now
```

Comment:
I think there is an issue in this line. It is getting the 'last_updated' in "def run" which means it is getting the "last_updated" from the updated index as this action runs after the 'index content' action. This means, we will never get any errata notification because it won't be any beyond the "last_updated".   It should retrieve the "last_updated" in "def plan" just like the older version of Katello (Satellite 6.9)